src/Core/Loading/ILoader.cs

```diff
@@ -36,68 +36,86 @@ public interface ILoader
         /// </summary>
         string? DefaultToFormat { get; set; }
 
+        /// <summary>
+        /// Loads the file specified by <paramref name="imageLocation"/>. If it is a Reko project
+        /// file, loads that. If not, tries to match the file contents to one of the known
+        /// file formats, or uses an explicity provided <paramref name="loader"/>.
+        /// </summary>
+        /// <param name="imageLocation"></param>
+        /// <param name="loader"></param>
+        /// <param name="addrLoad"></param>
+        /// <returns>Either a <see cref="ILoadedImage"/> instance, or null if the file format
+        /// wasn't recognized.</returns>
+        ILoadedImage? Load(ImageLocation imageLocation, string? loader = null, Address? addrLoad = null);
```

Comment:
Now you return `Blob` if format is unknown. So return can be not-nullable

Reply:
Fixed.